Description: allows CFG_FULL_NAME/CFG_ALT_NAME modification from debian/rules. 
Modified by Marcos Fouces to adjust to upstream release 1.9.0.
Author: Kees Cook <kees@debian.org>, Marcos Fouces <marcos@debian.org>
Last-Update: 2022-08-07
Forwarded: no

--- a/src/params.h
+++ b/src/params.h
@@ -135,11 +135,6 @@
 /*
  * File names.
  */
-#ifdef __DJGPP__
-#define CFG_FULL_NAME			"$JOHN/john.ini"
-#else
-#define CFG_FULL_NAME			"$JOHN/john.conf"
-#endif
 #if JOHN_SYSTEMWIDE
 #ifdef __DJGPP__
 #define CFG_PRIVATE_FULL_NAME		JOHN_PRIVATE_HOME "/john.ini"
